src/status_im2/common/toasts/view.cljs

```diff
@@ -0,0 +1,98 @@
+(ns status-im2.common.toasts.view
+  (:require
+   [quo2.components.notifications.toast :as quo2.toast]
+   [react-native.core       :as rn]
+   [react-native.gesture    :as gesture]
+   [react-native.reanimated :as reanimated]
+   [reagent.core            :as reagent]
+   [status-im.utils.utils   :as utils.utils]
+   [status-im2.common.toasts.subs]
+   [utils.re-frame          :as re-frame]))
+
+(def ^:private slide-out-up-animation
+  (-> reanimated/slide-out-up-animation
+      .springify
+      (.damping 20)))
+
+(def ^:private slide-in-up-animation
+  (-> reanimated/slide-in-up-animation
+      .springify
+      (.damping 20)))
+
+(def ^:private linear-transition
+  (-> reanimated/linear-transition
+      .springify
+      (.damping 20)))
+
+(defn container
+  [id]
+  (let [toast-opts         (re-frame/sub [:toasts/toast id])
+        duration           (get toast-opts :duration 3000)
+        on-dismissed       (get toast-opts :on-dismissed identity)
+        dismissed-locally? (reagent/atom false)
```

Comment:
atom should be defined outside render function

Reply:
done